src/components/graphs/piegraph/colorShader.js

```diff
@@ -0,0 +1,29 @@
+function LightenDarkenColor(col, amt) {
+  var usePound = false;
+
+  if (col[0] === '#') {
+    col = col.slice(1);
+    usePound = true;
+  }
+
+  var num = parseInt(col, 16);
+
+  var r = (num >> 16) + amt;
```

Comment:
Are these variables (R,G,B) references to color values? I assume so but want to be sure.

Reply:
Yes